docs/admin/access.rst

```diff
@@ -168,15 +168,15 @@ to a containing project as well.
 
 .. note::
 
-   Only first rule which is set is being used. So if you set all
+   Only the first rule will be used. So if you set all of
    :guilabel:`Component list`, :guilabel:`Components` and :guilabel:`Project`,
-   only :guilabel:`Component list` is being applied.
+   only :guilabel:`Component list` is the only one applied.
 
-Additional step is performed if checking permission for the translation:
```

Comment:
what does "checking" mean here?

Reply:
Checking whether user has access to a translation.